cmd/launcher/doctor.go

```diff
@@ -50,7 +49,6 @@ var (
 
 	// Indented output for checkup results
 	info func(a ...interface{})
-	warn func(a ...interface{})
```

Comment:
Feels weird removing this stuff. It's probably not a problem, per se, but it makes it harder to make a base of things ready to use. :|

Would the linter be appeased if we put it in a test?

Reply:
I could live with it though. It _is_ unused after all

Reply:
I'm happy to flag these as unused and keep them